crates/sui/tests/full_node_tests.rs

```diff
@@ -393,6 +393,36 @@ async fn test_full_node_indexes() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+// Test for syncing a node to an authority that already has many txes.
+#[tokio::test]
+async fn test_full_node_cold_sync() -> Result<(), anyhow::Error> {
+    telemetry_subscribers::init_for_testing();
+
+    let (swarm, mut context, _) = setup_network_and_wallet().await?;
+
+    let (_, _, _, _) = transfer_coin(&mut context).await?;
+    let (_, _, _, _) = transfer_coin(&mut context).await?;
+    let (_, _, _, _) = transfer_coin(&mut context).await?;
+    let (_transfered_object, _sender, _receiver, digest) = transfer_coin(&mut context).await?;
+
+    sleep(Duration::from_millis(1000)).await;
```

Comment:
we should sleep after `wait_for_tx`, right?

Let's add a comment on why we sleep here

Reply:
This sleep is to make sure that the authorities are quiescent before we start up the node - if they aren't, its not really a cold sync.